ush/forecast_predet.sh

```diff
@@ -602,7 +602,6 @@ FV3_predet(){
         export e3="${NMEM_ENS}"
         ;;
       *)
-        echo "FATAL ERROR: Unknown NET ${NET}, unable to determine appropriate post files"
```

Comment:
Why remove the FATAL ERROR message?

Reply:
I probably shouldn't have.  I'll double check and likely reinstate.